xray/commands/audit/sca/npm/npm.go

```diff
@@ -28,10 +28,14 @@ func BuildDependencyTree(npmArgs []string) (dependencyTrees []*xrayUtils.GraphNo
 	if err != nil {
 		return
 	}
-	npmArgs = addIgnoreScriptsFlag(npmArgs)
-
+	treeDepsParam := biutils.NpmTreeDepListParam{
+		Args: addIgnoreScriptsFlag([]string{}),
+	}
+	if params != nil {
+		treeDepsParam = createTreeDepsParam(params)
+	}
```

Comment:
Let's create NpmTreeDepListParam only in createTreeDepsParam:
```suggestion
	treeDepsParam := createTreeDepsParam(params)
```